dev/com.ibm.websphere.appserver.features/visibility/public/jsonb-2.0/resources/l10n/com.ibm.websphere.appserver.jsonb-2.0.properties

```diff
@@ -0,0 +1,18 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The Java API for JSON Binding (JSON-B) feature provides a standard for \
```

Comment:
change "Java API" to "Jakarta API"

Reply:
That's an oversight.  We'll make the change.

Reply:
Done.